src/main/scala/com/mwz/sonar/scala/ScalaPlugin.scala

```diff
@@ -16,40 +16,50 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
-package com.sagacify.sonar.scala
+package com.mwz.sonar.scala
 
-import com.buransky.plugins.scoverage.measure.ScalaMetrics
-import com.buransky.plugins.scoverage.sensor.ScoverageSensor
+import com.mwz.sonar.scala.scoverage.{ScoverageMetrics, ScoverageSensor}
+import com.mwz.sonar.scala.sensor.ScalaSensor
+import com.mwz.sonar.scala.util.JavaOptionals._
 import com.ncredinburgh.sonar.scalastyle.{ScalastyleQualityProfile, ScalastyleRepository, ScalastyleSensor}
 import org.sonar.api.Plugin
 import org.sonar.api.config.Configuration
 import org.sonar.api.resources.AbstractLanguage
-
+import scalariform.ScalaVersions
 import scalariform.lexer.{ScalaLexer, Token}
 
 /**
  * Defines Scala as a language for SonarQube.
+ *
+ * @author mwz
```

Comment:
I'm not sure if there is any value in keeping the authors for each class as it's something that's going to change over time and it's just another thing that you are going to have to maintain, so I suppose if someone is really interested in knowing who wrote a particular piece of code they can always use `git blame` to find that out.

Reply:
Uhm, yeah, you had two valid points there.
I really should had asked you about it before.
I will remove them.